.github/workflows/asan/start.sh

```diff
@@ -111,6 +111,7 @@ rm -f "$WORK_DIR/ccache.tar.gz"
 
 
 export PRELOAD=$PWD/clang+llvm-9.0.0-x86_64-linux-gnu-ubuntu-16.04/lib/clang/9.0.0/lib/linux/libclang_rt.asan-x86_64.so
+export ASAN_SYMBOLIZER_PATH=$PWD/clang+llvm-9.0.0-x86_64-linux-gnu-ubuntu-16.04/bin/llvm-symbolizer
```

Comment:
Actually on second though it may be better to split this change out into a separate PR to us + GDAL.
May be easier to do this now instead of later, but your call (since you'll be the one doing the work).